### DISCUSSION TITLE
next/link and absolute URLs causing crashes on iOS14

### DISCUSSION THREAD

**Discussion by clam61** [vercel/next.js#48369] [Help]
### Summary

If I have something like this:

`<Link href={url}>sample link</Link>`

where the url = 'https://www....' I will get an error in iOS 14.1 with Safari--something about next/router ignoring "//" and "\" in urls.  My page crashes. The same code works fine on iOS 15 and above.

Do I have to use `<a>` or `<Link>` depending if the href value?

**Comment by 808vita** [comment]
Is it possible for you to post logs recorded in the browser console , try it with older chrome browser .
If some page crashes in old safari , it is highly likely that page will crash in older versions of chrome like 60- 70 version .
could be because of browser compatibility , try to extend it .
